components/input-number/__tests__/index.test.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { mount } from 'enzyme';
+import { ArrowUpOutlined,ArrowDownOutlined } from '@ant-design/icons';
```

Comment:
```suggestion
import { ArrowUpOutlined, ArrowDownOutlined } from '@ant-design/icons';
```

提交时不要 -n，走一下 prettier。